src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -1,18 +1,160 @@
+
+  private final SparkPIDController m_rollerPidController; // Roller velocity PID controllers
+  private final RelativeEncoder m_rollerEncoder; // Roller Relative Encoder
+
+  private final DigitalInput m_intakeSensor; // Line break note sensor
+
+  private boolean m_testModeCheck1; // Booleans for if test mode is enabled
+  private boolean m_testModeCheck2;
+
+  // Constructs intake and initializes motor, PID, encoder objects, sensor
+  public Intake() {
+    m_rollerFollowerMotor = new CANSparkMax(IntakeConfig.kFollowerMotorID, MotorType.kBrushless);
+    m_rollerLeaderMotor = new CANSparkMax(IntakeConfig.kLeaderMotorID, MotorType.kBrushless);
+    m_rollerFollowerMotor.follow(this.m_rollerLeaderMotor);
+
+    m_rollerPidController = m_rollerLeaderMotor.getPIDController();
+    m_rollerEncoder = m_rollerLeaderMotor.getEncoder();
+    zeroEncoders();
+
+    m_intakeSensor = new DigitalInput(RobotConstants.IntakeConfig.kLineBreakSensor);
+
+    // set Roller PID coefficients
+    m_rollerPidController.setP(RobotConfig.IntakeConfig.kRollerP);
+    m_rollerPidController.setI(RobotConfig.IntakeConfig.kRollerI);
+    m_rollerPidController.setD(RobotConfig.IntakeConfig.kRollerD);
+    m_rollerPidController.setFF(RobotConfig.IntakeConfig.kRollerFF);
+    m_rollerPidController.setIZone(RobotConfig.IntakeConfig.kRollerIZone);
+    m_rollerPidController.setOutputRange(
+        RobotConfig.IntakeConfig.kRollerMinOutput, RobotConfig.IntakeConfig.kRollerMaxOutput);
+
+    // display Roller PID coefficients on SmartDashboard & test mode booleans
+    SmartDashboard.putNumber(
+        RobotConfig.IntakeConfig.kRollerPGainKey, RobotConfig.IntakeConfig.kRollerP);
+    SmartDashboard.putNumber(
+        RobotConfig.IntakeConfig.kRollerIGainKey, RobotConfig.IntakeConfig.kRollerI);
+    SmartDashboard.putNumber(
+        RobotConfig.IntakeConfig.kRollerDGainKey, RobotConfig.IntakeConfig.kRollerD);
+    SmartDashboard.putNumber(
+        RobotConfig.IntakeConfig.kRollerFFGainKey, RobotConfig.IntakeConfig.kRollerFF);
+    SmartDashboard.putNumber(
+        RobotConfig.IntakeConfig.kRollerIZoneKey, RobotConfig.IntakeConfig.kRollerIZone);
+    SmartDashboard.putNumber(
+        RobotConfig.IntakeConfig.kRollerMinOutputKey, RobotConfig.IntakeConfig.kRollerMinOutput);
+    SmartDashboard.putNumber(
+        RobotConfig.IntakeConfig.kRollerMaxOutputKey, RobotConfig.IntakeConfig.kRollerMaxOutput);
+    SmartDashboard.putBoolean(RobotConfig.IntakeConfig.kTestCheck1Key, m_testModeCheck1);
+    SmartDashboard.putBoolean(RobotConfig.IntakeConfig.kTestCheck2Key, m_testModeCheck2);
+  }
 
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
+
+    boolean testCheck1 = SmartDashboard.getBoolean(RobotConfig.IntakeConfig.kTestCheck1Key, false);
+    boolean testCheck2 = SmartDashboard.getBoolean(RobotConfig.IntakeConfig.kTestCheck2Key, false);
+
+    // Checks if test mode is enabled
```

Comment:
What's the purpose of these?

Reply:
I recommended them for test mode for PID constant setting so they aren't accidentally loaded or something, but I realized `testPeriodic()` might be a better function for them. Either way, since we aren't doing PID on the intake, they are unnecessary.

Reply:
They have been removed